weasel/tests/test_cli_app.py

```diff
@@ -0,0 +1,150 @@
+from pathlib import Path
+
+import pytest
+import srsly
+from typer.testing import CliRunner
+
+from weasel._util import app, get_git_version
+
+
+def has_git():
+    try:
+        get_git_version()
+        return True
+    except RuntimeError:
+        return False
+
+
+# project tests
+
+SAMPLE_PROJECT = {
+    "title": "Sample project",
+    "description": "This is a project for testing",
+    "assets": [
+        {
+            "dest": "assets/spacy-readme.md",
+            "url": "https://github.com/explosion/spaCy/raw/dec81508d28b47f09a06203c472b37f00db6c869/README.md",
+            "checksum": "411b2c89ccf34288fae8ed126bf652f7",
+        },
+        {
+            "dest": "assets/citation.cff",
+            "url": "https://github.com/explosion/spaCy/raw/master/CITATION.cff",
+            "checksum": "c996bfd80202d480eb2e592369714e5e",
```

Comment:
Silly little thing, but let's have these tests refer to other files and not spaCy-repo-specific ones. Could be weasel's readme, for instance.

Reply:
Nice catch!